src/controller/CHIPDeviceController.h

```diff
@@ -1120,6 +1120,11 @@ class DLL_EXPORT DeviceCommissioner : public DeviceController,
     Credentials::AttestationVerificationResult mAttestationResult;
     Platform::UniquePtr<Credentials::DeviceAttestationVerifier::AttestationDeviceInfo> mAttestationDeviceInfo;
     Credentials::DeviceAttestationVerifier * mDeviceAttestationVerifier = nullptr;
+
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+    chip::Credentials::P256PublicKeySpan jfAdminICACPKSpan;
```

Comment:
This looks incredibly unsafe.  Where is this span pointing into?  What ensures that thing survives long enough?

Having the  `chip::Optional<chip::Crypto::P256PublicKey>` as a member, which is what was here before, was absolutely the right thing to do.  This setup is asking for a use-after-free security bug.